### PR TITLE
Fix manual alignment hinting

### PR DESCRIPTION
**Pull request by dbacomputer**
## Summary

This PR fixes two issues found while reviewing canvas alignment guides.

Screenshot 2026-08-05 at 8 22 35 AM
Screenshot 2026-08-05 at 8 22 32 AM

### 1. Restore the three-line alignment hint

- Equal-size aligned nodes once again show their left edge, center, and right edge guides.
- Magnetic placement still uses one deterministic winning candidate per axis, so snapping remains stable and unchanged.
- The additional lines are visual confirmation derived from the final snapped geometry.

### 2. Fix Shift-click multi-selection

- Shift-clicking a second node now preserves the first selection and adds the second node.
- Selected nodes can then be dragged together as a group.
- The multi-select story uses the click event's actual Shift modifier state instead of relying solely on React Flow's iframe-level key tracking.

## Root cause

During PR #956's finalization, snapping and visual guide selection were coupled around one winning candidate per axis. That correctly made snapping deterministic, but unintentionally reduced the approved left/center/right visual treatment to one guide.

The multi-select story also relied on React Flow's global modifier-key state. In Storybook's iframe, that state was not reliably active when node selection occurred, causing the second Shift-click to replace rather than extend the selection.

## User impact

Users receive the intended three-line visual confirmation without any change to final snapped placement. The multi-select example now behaves as documented, allowing multiple nodes to be selected with Shift-click and dragged together.

## Validation

- `pnpm --filter @uipath/apollo-react exec vitest --run src/canvas/components/AlignmentGuides/useAlignmentGuides.test.ts` — 10 tests passed
- `pnpm --filter @uipath/apollo-react format:check` — passed
- Verified the Alignment Guides story locally
- Verified Shift-click retains both selections in Multi-select Drag
- Verified selected nodes drag together

### CODE REVIEW

File: packages/apollo-react/src/canvas/components/AlignmentGuides/useAlignmentGuides.ts [hunk 1]
```diff
@@ -92,38 +92,47 @@ function resolveAxisCandidate(
...
+      orientation === 'vertical'
+        ? ([bounds.y1, bounds.y2] as const)
+        : ([bounds.x1, bounds.x2] as const)
+    );
+
+    return [
+      {
+        id: `${orientation}-${position}`,
+        orientation,
+        position,
```
[the remark this answers is not in the document]
Comment (dbacomputer) [reply packages/apollo-react/src/canvas/components/AlignmentGuides/useAlignmentGuides.ts:121]: Fixed in dcc8650f. Guide IDs now include the dragged anchor index, so coincident edge/center anchors remain unique for zero-width or zero-height bounds. I also added a regression test covering zero-size bounds and asserting all six generated guide IDs are unique; the focused suite passes 11/11.

**Review by CalinaCristian — APPROVED** [review]
